### ISSUE TITLE
Cargo.toml missing on gh-pages branch

### ISSUE THREAD

**Issue by maciek134** [DerekStride/tree-sitter-sql#241]
Hey, thanks for working on this!

Since you don't publish the parser on crates.io I wanted to use a git link to use it in a Rust project, but the `gh-pages` branch doesn't contain `Cargo.toml` resulting in an error during building:

```
error: failed to get `tree-sitter-sql` as a dependency of package `snip`

Caused by:
  failed to load source for dependency `tree-sitter-sql`

Caused by:
  Unable to update https://github.com/DerekStride/tree-sitter-sql.git?branch=gh-pages

Caused by:
  Could not find Cargo.toml in `~/.cargo/git/checkouts/tree-sitter-sql-4edcb56b7fb1716a/da2d1ef`

```

Are you open to adding it? Or maybe even publishing the parser on crates.io?

**Comment by matthias-Q** [comment]
@DerekStride looks like the gh-publish Action did not run successfully

**Comment by DerekStride** [comment]
Opps, Fix up in #243
